experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -80,7 +80,9 @@ export class FetchInstrumentation extends InstrumentationBase<FetchInstrumentati
   private _tasksCount = 0;
 
   constructor(config: FetchInstrumentationConfig = {}) {
-    super('@opentelemetry/instrumentation-fetch', VERSION, config);
+    super('@opentelemetry/instrumentation-fetch', VERSION, config, {
+      description: PACKAGE_DESCRIPTION,
```

Comment:
I wonder if this makes sense for instrumentations that target the browser. It adds to the bundle size. We can also not tree-shake it out even though it's never really used.

Reply:
as a reference, collector repo [define `metadata.yaml` files per component](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlpexporter/metadata.yaml) which include stuff like this:

```yaml
type: otlp

status:
  class: exporter
  stability:
    stable: [traces, metrics]
    beta: [logs]
  distributions: [core, contrib]
```

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/4725